packages/contracts/contracts/PriceFeed.sol

```diff
@@ -91,6 +91,10 @@ contract PriceFeed is BaseMath, IPriceFeed, AuthNoOwner {
 
     // --- Functions ---
 
+    function latestRound() external view returns (uint80) {
```

Comment:
Q: This is introduced but never used, why?

Reply:
removed